collectd-marathon/README.md

```diff
@@ -69,7 +69,9 @@ Using the sample configuration file [20-collectd-marathon.conf](https://github.c
 | Import | Path to the name of the pythom module with out the .py extension | `marathon` |
```

Comment:
This is a mistake I made.  Could you please correct the word "pythom" to "python".